articles/cross-origin-authentication/fingerprinting.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Fingerprinting of Username + Password Login Requests
+description: An explanation of the technique used to mitigate CSRF attacks for cross-origin authentication requests.
+---
+# Fingerprinting of Username + Password Login Requests
+
+Auth0 has announced a [vulnerability](/migrations#introducing-lock-v11-and-auth0-js-v9) in all versions of Lock (<11) and auth0.js (<9). We strongly recommend that all users of legacy versions migrate to the newest, most secure versions immediately. Support for conducting logins with a username and password with these versions is now end of life and deprecated versions will stop working on **July 16th, 2018**.
```

Comment:
Should [vulnerability] this link to the blog post?

Reply:
Yeah, good idea now that it's published, will update